x-pack/plugins/threat_intelligence/public/modules/indicators/components/query_bar/query_bar.test.tsx

```diff
@@ -13,20 +13,24 @@ import userEvent from '@testing-library/user-event';
 import { FilterManager } from '@kbn/data-plugin/public';
 
 import { coreMock } from '@kbn/core/public/mocks';
-import { TestProvidersComponent } from '../../../../common/mocks/test_providers';
+import { TestProvidersComponent, unifiedSearch } from '../../../../common/mocks/test_providers';
 import { getByTestSubj } from '../../../../../common/test/utils';
+import { setAutocomplete } from '@kbn/unified-search-plugin/public/services';
 
 const mockUiSettingsForFilterManager = coreMock.createStart().uiSettings;
 
 const filterManager = new FilterManager(mockUiSettingsForFilterManager);
 
-// Failing: See https://github.com/elastic/kibana/issues/137752
-describe.skip('QueryBar ', () => {
+describe('QueryBar ', () => {
   const onSubmitQuery = jest.fn();
   const onSubmitDateRange = jest.fn();
   const onSavedQuery = jest.fn();
   const onChangedQuery = jest.fn();
 
+  beforeEach(() => {
+    setAutocomplete(unifiedSearch.autocomplete);
```

Comment:
I guess this is the fix for the error in [here](https://buildkite.com/elastic/kibana-on-merge/builds/19226#01825e08-139a-4749-9785-012d4213f658/8483-8486) ?

Reply:
Yes, indeed. Unified search uses some singleton cache that is not populated & somehow makes a difference on CI. I cannot reproduce this locally though.

Reply:
I will reopen this targeting `main`, sorry for inconvenience.